lib/metaforce/login.rb

```diff
@@ -22,7 +22,12 @@ def login
           :password => password
         }
       end
-      response.body[:loginResponse][:result]
+
+      # The global override to convert the SOAP XML to camel case breaks the
```

Comment:
That was added in `lib/metaforce.rb` in https://github.com/Yesware/metaforce/commit/358f7a43bffeb04ecde0fc963f80a41b84d083c3.  I talked to @tjwp-yesware and he didn't recall exactly why he needed to do that.  I thought it was safer to make a single exception than to undo an explicit change; it was probably done for a good reason.